### PR TITLE
Add node to cluster

### PR DESCRIPTION
**Pull request by stgleb**
Implement add node to cluster that was previously
deployed

### CODE REVIEW

File: pkg/kube/handler.go [hunk 1]
```diff
@@ -208,3 +225,135 @@ func (h *Handler) getCerts(w http.ResponseWriter, r *http.Request) {
...
+	}
+
+	writer, err := getWriter(t.ID)
+
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	errChan := t.Run(context.Background(), *config, writer)
```
Comment (egopher) [inline pkg/kube/handler.go]: Better use context from the request.

Comment (stgleb) [reply pkg/kube/handler.go]: Not in that case, since request will be finished much earlier that task.

File: pkg/kube/handler.go [hunk 2]
```diff
@@ -208,3 +225,135 @@ func (h *Handler) getCerts(w http.ResponseWriter, r *http.Request) {
...
+	// Respond to client side that request has been accepted
+	w.WriteHeader(http.StatusAccepted)
+	err = json.NewEncoder(w).Encode(&workflows.TaskResponse{
+		ID: t.ID,
```
Comment (egopher) [inline pkg/kube/handler.go]: Is it possible to add an URL that could be polled?

Comment (stgleb) [reply pkg/kube/handler.go]: Sure, as a good rest api is will be nice to a dd links field

File: pkg/kube/handler.go [hunk 3]
```diff
@@ -208,3 +229,111 @@ func (h *Handler) getCerts(w http.ResponseWriter, r *http.Request) {
...
+		http.NotFound(w, r)
+		return
+	}
+
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	kubeProfile := profile.Profile{
```
Comment (egopher) [inline pkg/kube/handler.go]: Do we really need all this info? I mean cluster is already created.

Comment (stgleb) [reply pkg/kube/handler.go]: Yes, I need all these settings because simple node creation needs all of these.

**Pull request merged**